tests/ignite/contrib/handlers/test_lr_finder.py

```diff
@@ -336,9 +336,9 @@ def test_lr_suggestion_multiple_param_groups(lr_finder):
     import numpy as np
 
     noise = 0.06
-    lr_finder._history["loss"] = torch.tensor(np.linspace(-5.0, 5, num=50) ** 2 + noise)
+    lr_finder._history["loss"] = torch.tensor(np.linspace(-5.0, 5, num=50) ** 2 + noise, dtype=torch.float)
```

Comment:
Why can't we use `torch.linspace` here ?

Reply:
Because of creating the loss manually and using `np.linspace`, this produces arrays of type `double`, and that causes this error 
```
min_loss_idx = torch.FloatTensor(loss).argmin()
E       TypeError: expected Float (got Double)

```